roles/docker-storage-setup/tasks/main.yaml

```diff
@@ -25,5 +25,21 @@
     - ansible_distribution_version | version_compare('7.4', '<')
     - ansible_distribution == "RedHat"
 
+- block:
+    - name: create the docker-storage-setup config file for CentOS
+      template:
+        src: "{{ role_path }}/templates/docker-storage-setup-dm.j2"
+        dest: /etc/sysconfig/docker-storage-setup
+        owner: root
+        group: root
+        mode: 0644
+
+  # TODO(shadower): Find out which CentOS version supports overlayfs2
```

Comment:
FWIW, overlay2 works with the centos 7.3 in tripleo upstream CI, with the latest kernel installed

Reply:
For RHEL I'd say to be careful about supportability starting with 7.4[1]

[1] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/managing_storage_with_docker_formatted_containers#overlay_graph_driver